modules/storages/app/models/storages/one_drive_storage.rb

```diff
@@ -54,5 +58,13 @@ def uri
     def connect_src
       %w[https://*.sharepoint.com https://*.up.1drv.com]
     end
+
+    def automatic_management_unspecified?
+      automatic_management_enabled.nil?
+    end
```

Comment:
🟠 Should be Ok to move this method up the chain?

Reply:
True, will do in a bit